tiatoolbox/annotation/storage.py

```diff
@@ -1843,6 +1853,40 @@ def bquery(
             }
         return {key: bounds for key, *bounds in cur.fetchall()}
 
+    @staticmethod
+    def __handle_callable_pquery(unique, select, cur, where):
+        """Handles Callable pquery."""
+        if unique:
+            return {
+                select(json.loads(properties))
+                for properties, in cur.fetchall()
+                if where(json.loads(properties))
+            }
+        return {
+            key: select(json.loads(properties))
+            for key, properties in cur.fetchall()
+            if where(json.loads(properties))
+        }
+
+    @staticmethod
+    def __check_pquery_type(select, where):
```

Comment:
Should be single underscore

Reply:
Resolved in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/330/commits/82155397526452234dc3fbbfa80a77e520506d13